src/drafts/SelectPanel2/SelectPanel.stories.tsx

```diff
@@ -471,6 +472,149 @@ export const TODO3NoCustomisation = () => {
+        onCancel={() => {
+          // eslint-disable-next-line no-console
+          console.log('panel was closed')
+        }}
+      >
+        {/* TODO: the ref types don't match here, use useProvidedRefOrCreate */}
+        {/* @ts-ignore todo */}
+        <SelectPanel.Button leadingIcon={GitBranchIcon} trailingIcon={TriangleDownIcon}>
+          main
+        </SelectPanel.Button>
+
+        <SelectPanel.Header>
+          <SelectPanel.Heading>Switch branches/tags</SelectPanel.Heading>
+          <SelectPanel.SearchInput onChange={onSearchInputChange} sx={{marginBottom: 2}} />
+
+          <Box id="filters" sx={{display: 'flex'}}>
+            <Button
+              variant="invisible"
+              sx={{fontWeight: selectedFilter === 'tags' ? 'semibold' : 'normal', color: 'fg.default'}}
+              onClick={() => setSelectedFilter('branches')}
+            >
+              Branches <Button.Counter>{20}</Button.Counter>
+            </Button>
+            <Button
+              variant="invisible"
+              sx={{fontWeight: selectedFilter === 'tags' ? 'semibold' : 'normal', color: 'fg.default'}}
+              onClick={() => setSelectedFilter('tags')}
+            >
+              Tags <Button.Counter>{8}</Button.Counter>
+            </Button>
+          </Box>
+        </SelectPanel.Header>
+
+        <SelectPanel.ActionList selectionVariant="single">
+          {/* slightly different view for search results view and list view */}
+          {query ? (
+            filteredLabels.length > 1 ? (
+              filteredLabels.map(label => (
+                <ActionList.Item
+                  key={label.id}
+                  onSelect={() => onLabelSelect(label.id)}
+                  selected={selectedLabelIds.includes(label.id)}
+                >
+                  {label.name}
+                  <ActionList.TrailingVisual>{label.trailingInfo}</ActionList.TrailingVisual>
+                </ActionList.Item>
+              ))
+            ) : (
+              <SelectPanel.EmptyMessage>No labels found for &quot;{query}&quot;</SelectPanel.EmptyMessage>
+            )
+          ) : (
+            <>
+              {data[selectedFilter].sort(sortingFn).map(item => {
+                return (
+                  <>
+                    <ActionList.Item
+                      key={item.id}
+                      onSelect={() => onLabelSelect(item.id)}
+                      selected={selectedLabelIds.includes(item.id)}
+                    >
+                      {item.name}
+                      <ActionList.TrailingVisual>{item.trailingInfo}</ActionList.TrailingVisual>
+                    </ActionList.Item>
+                  </>
+                )
+              })}
+            </>
+          )}
+        </SelectPanel.ActionList>
+        <SelectPanel.Footer>
+          {/* TODO: Can't disable Cancel and Save yet */}
+          <SelectPanel.SecondaryButton as="a" href="/branches">
+            See all branches
```

Comment:
In this particular example, we want to show a different secondary link depending on the selected tab ("View all branches" or "View all tags")